docs/annexes/annex-06-pid-rulebook.md

```diff
@@ -491,16 +517,110 @@ If other CDDL representation types will be used in future versions of
 this document, the corresponding rules for encoding them with JSON will
 be added here.
 
-##### 2.5.3.2 Further stipulations
-
-Data elements in Table 1 and Table 2 SHALL be released only in a VP
-Token, as specified in \[OpenID4VP\]. This means that these data
-elements SHALL be signed by the PID Provider, not by the Wallet
-Instance.
+##### 2.6.3.2 Data Element Identifer to Claim Mapping
+
+SD-JWT encoded PID attestations SHALL use claim names registered in the JSON Web Token Claims Registry \[IANA-JWT-Claims\]. Table 3 below maps the data element identifiers from Table 1 and Table 2 above to the corresponding claim names. 
+
+Note that a hierarchical claim name structure can be used in SD-JWT VC encoded PID attestations as SD-JWT allows for individual selective disclosure of objects and their properties. A hierarchical claim name structure is indicated by the notation `parent/child` in Table 3.
+
+Data element identifiers not listed in Table 3 are used as defined above.
+
+| Data Element Identifier | Claim Name | Note |
+|---|---|---|
+| `birth_date` | `birthdate` |  |
+| `age_over_18` | `age_equal_or_over/18` | `18` property name is a string. *)|
+| `age_over_NN` | `age_equal_or_over/NN` | NN property name is a string. *) |
+| `family_name_birth` | `birth_family_name` | |
+| `given_name_birth` | `birth_given_name` | |
+| `birth_place` | `place_of_birth/locality` | *) |
```

Comment:
`birth_place` and `birth_city` are both mapped to `place_of_birth/locality`

Is this correct?

Reply:
Have we truly identified a use case that necessitates the `birth_city` field when we already have `birth_place`?

Are there legal documents that simultaneously capture both the city and another place, and if so, is the place in this case more or less generic than the city (part of the city or region the city belongs to)? 

If not, it appears that the `birth_city` field is redundant since `birth_place` could adequately contain its value.

Removing the `birth_city` field aligns this table with IANA's `place_of_birth` definition (`locality`). 
* [IANA JWT Claims](https://www.iana.org/assignments/jwt/jwt.xhtml)
* [OpenID Connect Identity Assurance Claims](https://openid.net/specs/openid-connect-4-identity-assurance-1_0-13.html#claims)

Reply:
That would require changes to 5.1.1.2 first which seems to be outside of the scope of this PR (but I do believe would be a good idea)

Reply:
I agree and would support a separate PR for that.

Reply:
Can I ask what is the outcome of this?

The table still contains `birth_place` and `birth_city`, both mapped to `place_of_birth.locality`.